std/experimental/ndslice/selection.d

```diff
@@ -0,0 +1,1386 @@
+/**
+$(SCRIPT inhibitQuickIndex = 1;)
+
+Селекторы создают новый тип итератора для тех же данных.
+Сами данные остаются неизменными (to translate).
+
+$(H2 Subspace selectors)
+
+The destination of subspace selectors is painless generalization and combination of other selectors.
+`pack!K` creates a slice of slices `Slice!(N-K, Slice!(K+1, Range))` by packing last `K` dimensions of highest pack of dimensions,
+so type of element of `slice.byElement` is `Slice!(K, Range)`.
+Another way to use `pack` is transposition of packs of dimensions using `evertPack`.
```

Comment:
Those last three lines are missing so many articles* ("the", "an" etc.) I'm confused as to what it means. If you explain it I'd be happy to suggest a good wording.

*I struggled with not having articles when trying to learn Russian, I guess the problem goes both ways.